packages/css/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "cspell-dict-css",
   "version": "1.0.1",
-  "description": "CSS dictionary for cspell. -- Private until verified",
+  "description": "CSS dictionary for cspell.",
   "private": true,
   "bin": {
     "cspell-dict-css-link": "./link.js",
     "cspell-dict-css-unlink": "./unlink.js"
   },
   "scripts": {
-    "build": "cross-env NODE_OPTIONS=--max_old_space_size=8192 cspell-tools compile \"css.txt\" -o .",
```

Comment:
Can probably remove cross-env as a devDependency then. I had to add it since it was part of the generator template